Mathlib/CategoryTheory/Opposites.lean

```diff
@@ -160,6 +160,14 @@ protected def op (F : C ⥤ D) : Cᵒᵖ ⥤ Dᵒᵖ where
   obj X := op (F.obj (unop X))
   map f := (F.map f.unop).op
 
+/-- The functor which assigns to a functor its opposite -/
+def opFunctor  :  (C ⥤ D)ᵒᵖ ⥤ (Cᵒᵖ ⥤ Dᵒᵖ)  where
```

Comment:
This is a duplicate of `Functor.opHom` that is defined further below in the same file.

Reply:
Ah ! I'll close this then..